htdocs/js/jquery.fileupload.js

```diff
@@ -209,10 +213,13 @@ $(function() {
         $.extend( image, data );
         $field.data( "image-attributes", image );
 
+        var escape_titletext = image.title.replace( /</g, '&lt;' ).replace( /'/g, "&apos;" );
```

Comment:
I really don't know jQuery well, but this does work.  I'm open to a more elegant solution if anyone has one.